src/backend/core/services/ai_services.py

```diff
@@ -66,11 +65,7 @@ def call_ai_api(self, system_content, text):
             ],
         )
 
-        content = response.choices[0].message.content
-        sanitized_content = re.sub(r"(?<!\\)\n", "\\\\n", content)
-        sanitized_content = re.sub(r"(?<!\\)\t", "\\\\t", sanitized_content)
-
-        json_response = json.loads(sanitized_content)
+        json_response = json.loads(response.choices[0].message.content)
```

Comment:
Your reasoning is solid, and if the fix is functional and solves the problem effectively, it seems reasonable to move forward without additional complexity. 

However, we could enhance the implementation further with a try except, ignore if the sanitization fails and fallback on the `json.loads`

Reply:
Yes better to do that.